api/tx_args.go

```diff
@@ -268,10 +273,42 @@ func (args *SendTxArgs) toTransaction() (*types.Transaction, error) {
 			input = *args.Payload
 		}
 
+		if args.Recipient == nil && len(input) == 0 {
+			return nil, errTxArgNilContractData
+		}
+
+		al := types.AccessList{}
+		if args.AccessList != nil {
+			al = *args.AccessList
+		}
+		if *args.TypeInt == types.TxTypeEthereumAccessList {
+			return types.NewTx(&types.TxInternalDataEthereumAccessList{
```

Comment:
How about updating `genTxValuesMap` & `setDefaults` to support new types? Then, we don't need to handle new types specially here.

Reply:
Can you check [this commit](https://github.com/klaytn/klaytn/pull/1213/commits/b1fa9fee698a62059cc66df9f63ba80cad3143eb)?